app/js/lib/foxbox/recipes.js

```diff
@@ -136,7 +136,7 @@ export default class Recipes {
 
   getGetters() {
     // Currently we support only Clock and Motion Sensor as triggers.
-    const supportedKinds = ['CurrentTimeOfDay', 'OpenClosed'];
+    const supportedKinds = ['CurrentTimeOfDay', 'OpenClosed', 'LightOn'];
```

Comment:
@gmarty mm does it mean we'll have getter without any options/values in ```IF``` section in the recipe page?